pandas/io/tests/parser/python_parser_only.py

```diff
@@ -171,3 +171,18 @@ def test_read_table_buglet_4x_multiindex(self):
             columns=list('abcABC'), index=list('abc'))
         actual = self.read_table(StringIO(data), sep='\s+')
         tm.assert_frame_equal(actual, expected)
+
+    def test_temporary_file(self):
+        # GH13398
+        data1 = "0,0"
+        data2 = data1.replace(",", " ")
+
+        from tempfile import TemporaryFile
+        new_file = TemporaryFile("w+")
+        new_file.write(data2)
+        new_file.flush()
+        new_file.seek(0)
+
+        result = self.read_csv(new_file, sep=r"\s+", header=None)
```

Comment:
Can you double check that this actually errors without your changes?  I couldn't get an error on my end, which means you need to change the `sep` argument.